src/controllers/helpcontroller.js

```diff
@@ -0,0 +1,55 @@
+class HelpController extends Controller
+{
+    startIntro()
+    {
+        var options = {
```

Comment:
Options can be set in the `config` file (I'd suggest a field named `tutorial`) - similar to how toastr is configured.